pipeline/dbt/models/intermediate/sources/dora/int_dora__contacts.sql

```diff
@@ -0,0 +1,19 @@
+WITH structure_contacts AS (
+    SELECT courriel
+    FROM {{ ref('stg_dora__structures') }}
+    WHERE courriel != ''
+),
+
+service_contacts AS (
+    SELECT courriel
+    FROM {{ ref('stg_dora__services') }}
+    WHERE courriel != ''
```

Comment:
ça ne devrait pas être nécessaire à ce stade. Je m'efforce de faire ce genre de nettoyage en staging : 

https://github.com/gip-inclusion/data-inclusion/blob/main/pipeline/dbt/models/staging/sources/dora/stg_dora__services.sql#L51-L53

Reply:
OK mais il me faudra quand meme un `is not None`. 

Question subsidiaire : du coup je suppose que je retire le test qui vérifie l'égalité à `""` dans la table cible ?